module/signature/threshold.go

```diff
@@ -1,22 +1,5 @@
-// +build relic
-
 package signature
 
-import (
-	"fmt"
-
-	"github.com/onflow/flow-go/crypto"
-	"github.com/onflow/flow-go/crypto/hash"
-)
-
-// ThresholdVerifier is a verifier capable of verifying threshold signature
-// shares and verifying a threshold signature against a group public key.
-// *Important*: the threshold verifier can only verify signatures for
-// messages of one specific context (specified via the KMAC tag at construction).
-type ThresholdVerifier struct {
-	hasher hash.Hasher
-}
-
 // RandomBeaconThreshold returns the threshold (t) to allow the largest number of
```

Comment:
This method is still needed, because it's still [being used by DKG](https://github.com/onflow/flow-go/blob/feature/consensus-voting/cmd/bootstrap/dkg/dkg.go#L59)